resources/dicts/thoughts/alive/newborn.json

```diff
@@ -0,0 +1,31 @@
+[
+    {
+        "id": "gen_newborn",
+        "thoughts": [
+            "Mewls pitifully for milk",
+            "Tries to figure out all of the smells around them",
+            "Snoozes peacefully",
+            "Tries to open their eyes, but it's all blurry"
+        ],
+        "main_age_constraint": [
```

Comment:
Is it really main_age_constraint ? (can be)
But everywhere else the 'main_status_constraint' is used, maybe use this constraint to be more 'in line' with the overall structure?